packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/test.sh

```diff
@@ -16,6 +16,43 @@ test_val "$(compare_swing_store_export_data $EXPORT_DIR)" "match" "swing-store c
 rm -rf $EXPORT_DIR
 startAgd
 
+testHasChildren() {
+    path=$1
+    min=$2
+    line="$(agd query vstorage children $path -o jsonlines)"
+    ok=$(echo $line | jq ".children | length | . > $min")
+    test_val "$ok" "true" "$path: more than $min children"
+}
+
+# Check brand aux data for more than just vbank assets
+testHasChildren published.boardAux 3
+
+testDisplayInfo() {
+    name=$1
+    id=$2
+    expected=$3
+
+    line="$(agoric follow -lF :published.boardAux.$id -o jsonlines)"
+    displayInfo="$(echo $line | jq -c .displayInfo)"
+    test_val "$displayInfo" "$expected" "$name displayInfo from boardAux"
+}
+
+testDisplayInfo IST board0257 '{"assetKind":"nat","decimalPlaces":6}'
+
+testPurseValuePayload() {
+    addr=$1
+    wkAsset=$2
+    expected=$3
+
+    line="$(agoric follow -lF :published.wallet.$addr.current -o jsonlines)"
+    payload=$(echo $line | jq -c ".purses[] | select(.brand | contains(\"$wkAsset\")) | .balance.value.payload")
```

Comment:
we're identifying brands by their `allegedName` here. nasty hack. worth a comment, I expect

Reply:
Agreed, and also `jq` hygiene:
```suggestion
    payload=$(echo $line | jq --arg name "$wkAsset" -c ".purses[] | select(.brand | contains($name)) | .balance.value.payload")
```

Reply:
+1 marking hacks (e.g. `HACK`)

the jq hygiene I'm less concerned with because this will all be redone in pure JS after https://github.com/Agoric/agoric-sdk/pull/8237/